docs/dataformats/observation_lists.rst

```diff
@@ -10,38 +10,87 @@ the information in the run list. These filenames can either be given directly
 or there can be naming conventions, e.g. the event list for run ``42`` could be stored
 at ``$TEV_DATA/RUN_000042/Events.fits``.
 
+A run list must have at least a column called ``OBS_ID``::
+ 
+   OBS_ID
+   42
+   43
 
-CSV format
-----------
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is:
 
-We use the `CSV <http://en.wikipedia.org/wiki/Comma-separated_values>`_ (comma-separated values) format for run lists.
-This has the advantage that everyone can work with whatever tool they like. Here's some good options:
+================  ================================================================================================  =========
+column name       description                                                                                       required?
+================  ================================================================================================  =========
+OBS_ID            observation ID as an integer                                                                      yes
+RA                pointing position right ascension in equatorial (ICRS) coordinates                                yes?
+DEC               pointing position declination in equatorial (ICRS) coordinates                                    yes?
+AZ                average azimuth angle during the observation                                                      yes?
+ALT               average altitude angle during the observation                                                     yes?
+MUON_EFFICIENCY   average muon efficiency of the telescopes                                                         yes?
+TIME_START        start time of the observation stored as number of seconds after the time reference in the header  yes?
```

Comment:
Instead of writing "number of seconds after the time reference in the header" you should say MET.

And ideally `MET` would link to a "Glossary" section or a "Time in Gammapy" section.

@mapazarr – please add this and write a few sentences (linking to the Fermi description of time as done [here](https://hess-confluence.desy.de/confluence/display/HESS/HESS+Open+Source+Tools+-+Data+Formats+and+Exporters#HESSOpenSourceTools-DataFormatsandExporters-TimeinHESSdataanalysis)).
A "Time in Gammapy" section could go here: https://gammapy.readthedocs.org/en/latest/dataformats/index.html
A "Glossary" (for now only with one entry "MET") could go here: https://gammapy.readthedocs.org/en/latest/references.html
(make a sub-title "Papers" for what is there now and add the "Glossary" below).

I think this is not much work and doesn't need much discussion, i.e. you could add this in this PR and it would still be merged within ~ a day.